main.tf

```diff
@@ -88,9 +88,19 @@ locals {
         "awslogs-stream-prefix": "nginx"
```

Comment:
This should get changed to something like "helloworld". Keeping it "nginx" is pretty misleading.

Reply:
good catch